src/main/java/software/amazon/cloudformation/injection/AmazonWebServicesProvider.java

```diff
@@ -16,16 +16,35 @@
 
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
 import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.awscore.client.builder.AwsClientBuilder;
+import software.amazon.awssdk.awscore.client.builder.AwsSyncClientBuilder;
+import software.amazon.awssdk.core.client.config.ClientOverrideConfiguration;
+import software.amazon.awssdk.core.retry.RetryPolicy;
+import software.amazon.awssdk.http.SdkHttpClient;
 
 public abstract class AmazonWebServicesProvider {
 
+    protected static final ClientOverrideConfiguration CONFIGURATION = ClientOverrideConfiguration.builder()
```

Comment:
Given the available runtime, is 16 retries too many?

Reply:
I just ported it as is, but happy to tweak this if we have better values (the defaults?).

Reply:
+1,  default is 3 times I think (except 10 for DDB client). 16 seems too much.

Reply:
Also, since all aws clients are sharing the same HTTP client, do we want to increase the Maximum Connections?

Reply:
I think [50 is fine](https://github.com/aws/aws-sdk-java-v2/blob/45fe0672a5adea93d8f7e152a42b00c342636ae1/http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpConfigurationOption.java#L114), right? Given that the Lambda in each VM only really serves one request at a time (even if we create a few more connections for every request).